ProjectEureka/project-eureka-flutter
# PE-112 | Added support for HTTPS and Clean Imports+Code

**What I Did;**
- Changed to HTTPs for all services.
- Cleaned imports and code with Android Studio

**How to Test:**
1. Run `flutter pub upgrade`.
2. Change the .env `HOST` to what's provided along this PR in Discord.
3. Test everything and make sure stuff still works.
